package.json

```diff
@@ -79,6 +79,7 @@
     "react-native-screens": "^3.8.0",
     "react-native-sensitive-info": "^5.5.8",
     "react-native-share": "^7.1.2",
+    "react-native-splash-screen": "^3.2.0",
```

Comment:
Probably not a big deal now, but this library is not maintained anymore. People are using now [react-native-bootsplash](https://github.com/zoontek/react-native-bootsplash)

Reply:
oh sad! I did not know about that. let's see if any of this [differences](https://github.com/zoontek/react-native-bootsplash#%EF%B8%8F%EF%B8%8F-comparison-with-react-native-splash-screen) are important for us in future then I can change it.